src/templates/nulldata.js

```diff
@@ -14,15 +14,23 @@ function check (script) {
 check.toJSON = function () { return 'null data output' }
```

Comment:
That's a standard rule... I don't see any other standardness rules being enforced. (Unless I'm missing something)